zmon_aws_agent/common.py

```diff
@@ -1,5 +1,34 @@
+import time
+import logging
+
+from botocore.exceptions import ClientError
+
 from zmon_aws_agent import __version__
 
 
+MAX_RETRIES = 10
+
+logger = logging.getLogger(__name__)
+
+
 def get_user_agent():
     return 'zmon-aws-agent/{}'.format(__version__)
+
+
+def call_and_retry(fn, *args, **kwargs):
```

Comment:
maybe we can adapt this code to match what AWS suggests, see http://docs.aws.amazon.com/general/latest/gr/api-retries.html